app/components/tree_view_component.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: false
+
+class TreeViewComponent < ViewComponent::Base
+   include ApplicationHelper
+include ActionView::Helpers::TagHelper
+    
```

Comment:
try to remove 
```
include ApplicationHelper
include ActionView::Helpers::TagHelper
```

this component is really important, and need to be independent the most possible of any external behaviors (external dependecies).